mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_folder_security_health_analytics_custom_module_test.go.tmpl

```diff
@@ -162,6 +170,11 @@ resource "google_folder" "folder" {
   deletion_protection = false
 }
 
+resource "time_sleep" "wait_3_minute" {
```

Comment:
Is there a particular reason this uses 3m instead of 5m like the other tests?

Reply:
No, I thought it will take more time to wait since we are using 5m for other tests, but we can change it to 5m to maintain consistency and avoid suspicions regarding the timing differences of tests if in case any failure occurs in future.